src/CMakeLists.txt

```diff
@@ -101,6 +101,7 @@ set(algorithm_files
     nupic/algorithms/SpatialPooler.hpp
     nupic/algorithms/TemporalMemory.cpp
     nupic/algorithms/TemporalMemory.hpp
+    nupic/algorithms/ColumnPooler.cpp
```

Comment:
we'll need hpp in the long run

Reply:
I added this to the check-list at the top of the page, marking this as resolved for now, will fix later.